code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm

```diff
@@ -1,4 +1,4 @@
-/obj/structure/closet/secure_closet/guncabinet
```

Comment:
i dont think you meant to do this, should probably revert it

Reply:
Not seeing this missing on my end. Ill dobule check after latest commit